dbms/mysql/mysqldump/src/cloud/benchflow/collectors/dump.go

```diff
@@ -59,7 +59,6 @@ func backupHandler(w http.ResponseWriter, r *http.Request) {
 	ev := os.Getenv("TABLE_NAMES")
     tables := strings.Split(ev, ":")
```

Comment:
You should use comma (,) as separator for list and be consistent also for other collectors and monitors